src/app/components/normal-component/count-down/count-down.component.ts

```diff
@@ -113,6 +113,9 @@ export class CountDownComponent implements OnInit, OnDestroy {
         display(CountDownComponent.VERBOSE, this.debugName + '.pause(' + this.remainingMs + 'ms)');
 
         if (!this.started) {
+            // TODO FOR REVIEW: serait-il dégueulasse de centraliser nos "throw error"
+            // Ce afin d'en faire des "console.log" pour mieux voir QUAND dans les logs, c'est arrivé
+            console.log('Should not pause not started chrono (' + this.debugName + ')')
```

Comment:
oui pour centraliser les throw error, ou encore mieux: en supprimer la majorité !
(là on a déjà une partie qui est centralisée sous asserts, ici par exemple tu pourrais assert(this.started))